src/app/static/src/app/store/translations/locales.ts

```diff
@@ -12,6 +12,7 @@ export interface Translations {
     badUsernamePassword: string,
     bar: string,
     back: string,
+    blank: string,
```

Comment:
This might not be necessary, but v-translate was complaining when it received an empty string.